docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -855,7 +855,7 @@ This GatewayFilter replaces all headers with the given name, rather than adding.
 
 === SetStatus GatewayFilter Factory
 The SetStatus GatewayFilter Factory takes a single `status` parameter. It must be a valid Spring `HttpStatus`. It may be the integer value `404` or the string representation of the enumeration `NOT_FOUND`.
-
+The SetStatus GatewayFilter could be configured to set original Http Server code to response header
```

Comment:
Move this sentence to the section below where you added the example configuration.

I would reword it slightly

> The SetStatus GatewayFilter can be configured to return the original HTTP status code from the proxied request in a header in the response.  By default that header name is `original-http-status` but can be configured using the following properties.

Reply:
fixed